delocate/libsana.py

```diff
@@ -561,9 +699,9 @@ def wheel_libs(
     wheel_fname : str
         Filename of wheel
     filt_func : None or callable, optional
-        If None, inspect all files for library dependencies. If callable,
-        accepts filename as argument, returns True if we should inspect the
-        file, False otherwise.
+        If None, inspect all non-system files for library dependencies.
+        If callable, accepts filename as argument, returns True if we should
+        inspect the file, False otherwise.
```

Comment:
```suggestion
        inspect the file, False otherwise.
    ignore_missing : bool, default=False, optional
        If False, raise error for missing dependencies.  If True, continue without error.
```